src/flamenco/runtime/program/fd_system_program.c

```diff
@@ -683,7 +686,7 @@ fd_system_program_execute( fd_exec_instr_ctx_t ctx ) {
     break;
   }
   case fd_system_program_instruction_enum_create_account_with_seed: {
-    if( !VERIFY_SEED_UTF8( instruction.inner.create_account_with_seed.seed ) )
+    if( !instruction.inner.create_account_with_seed.seed || !VERIFY_SEED_UTF8( instruction.inner.create_account_with_seed.seed ) )
```

Comment:
This shouldn't be necessary at this point - the deserialization shouldn't allow this to be null (otherwise deserializing would fail)